tests/filecheck/dialects/omp/attrs.mlir

```diff
@@ -46,8 +46,44 @@
                 #omp<procbindkind spread>,
                 // CHECK-SAME: #omp<procbindkind spread>
                 
-                #omp<orderkind concurrent>
+                #omp<orderkind concurrent>,
                 // CHECK-SAME: #omp<orderkind concurrent>
 
+                #omp<device_type (any)>,
```

Comment:
Not a fan of all the `CHECK-SAME` in this file but I guess this is just following what is already there.